playground/src/components/state-machines-list/state-machines-list.js

```diff
@@ -90,9 +88,12 @@ export class StateMachinesList extends React.Component {
                     <Spacings.Inline alignItems="center">
                       <DotIcon size="small" theme="green" />
                       <Text.Detail isItalic={true}>
```

Comment:
Ha, for this we'd need `intlValues` ain't we?

Reply:
Yeah, I was going to open a PR to ui-kit 😅